tests/test_virtualenv.py

```diff
@@ -332,3 +339,44 @@ def test_relative_symlink(tmpdir):
     shutil.rmtree(workdir)
 
     assert os.path.exists(lib64)
+
+
+def test_missing_certifi_pem(tmp_path):
+    """Make sure that we can still create virtual environment if pip is
+    patched to not use certifi's cacert.pem and the file is removed.
+    This can happen if pip is packaged by Linux distributions."""
+    proj_dir = Path(__file__).parent.parent
+    support_original = proj_dir / "src" / "virtualenv_support"
+    pip_wheel = sorted(support_original.glob("pip*whl"))[0]
+    whl_name = pip_wheel.name
+
+    wheeldir = tmp_path / "wheels"
+    wheeldir.mkdir()
+    tmpcert = tmp_path / "tmpcert.pem"
+    cacert = "pip/_vendor/certifi/cacert.pem"
+    certifi = "pip/_vendor/certifi/core.py"
+    oldpath = b"os.path.join(f, 'cacert.pem')"
+    newpath = "r'{}'".format(tmpcert).encode("ascii")
```

Comment:
So the Windows path has backslashes and if you put `\u` into str, it blows up :grinning:

Reply:
easy to achieve by running in c:\users folder